docs/src/modules/components/TreeItem2Anatomy.js

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+
+export default function TreeItem2Anatomy() {
+  const { palette } = useTheme();
+
+  const src =
+    palette.mode === 'light'
+      ? '/static/x/tree-view-illustrations/tree-item-light.png'
```

Comment:
This pattern will blink for dark mode users (once docs-infra starts and finishes the migration to CSS variables). 

I think the right pattern here is to follow the other instances:
 https://github.com/mui/mui-x/blob/05e83ee9dfc7f0bdaf74703911cd7e175d37512e/docs/data/introduction/support/support.md?plain=1#L35-L40

Reply:
Another thought on this one. It feels like we should link the source to the Figma file we used so anyone can easy come here and change the image when we reorganize the component structure.